src/services/gameService.spec.ts

```
@@ -1,216 +1,188 @@
-        '--background-color': '#fff',
-        '--text-color': '#000',
-        '--accent-color': '#f00',
-        '--font-family': 'sans-serif',
-      },
-      startScreenImagePrompt: 'A test image prompt.',
-      conceptPrompt: 'A test concept prompt.',
-      aiSystemInstruction: 'You are a test AI.',
-    },
+    horrorIntensity: 0.1,
+    uiDistortion: { transform: 'none', filter: 'none', transition: 'none' },
+    genreConfig: mockGenreConfig,
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)